resources/resolver.go

```diff
@@ -84,22 +85,22 @@ func GetResourceEntries(typ ResourceType) ResourceEntryDefs {
 	case RESOURCETYPE_TRANSFORMERS:
 		return ResourceEntryDefs{
 			"config.json":             RESOURCE_REQUIRED,
-			"vocab.json":              RESOURCE_REQUIRED,
-			"merges.txt":              RESOURCE_REQUIRED,
+			"vocab.json":              RESOURCE_OPTIONAL,
+			"merges.txt":              RESOURCE_OPTIONAL,
 			"special_tokens_map.json": RESOURCE_OPTIONAL,
 			"unitrim.json":            RESOURCE_OPTIONAL,
 			"wordtokens.json":         RESOURCE_OPTIONAL,
 			"specials.txt":            RESOURCE_OPTIONAL | RESOURCE_DERIVED,
 			"tokenizer_config.json":   RESOURCE_OPTIONAL,
-			"tokenizer.json":          RESOURCE_OPTIONAL,
+			"tokenizer.json":          RESOURCE_REQUIRED,
 			"pytorch_model.bin":       RESOURCE_MODEL,
 		}
 	case RESOURCETYPE_DIFFUSERS:
 		return ResourceEntryDefs{
-			"feature_extractor/preprocessor_config.json": RESOURCE_OPTIONAL,
+			"feature_extractor/preprocessor_config.json": RESOURCE_REQUIRED,
 			"safety_checker/config.json":                 RESOURCE_OPTIONAL,
-			"safety_checker/pytorch_model.bin":           RESOURCE_OPTIONAL,
-			"scheduler/scheduler_config.json":            RESOURCE_REQUIRED,
+			"safety_checker/pytorch_model.json":          RESOURCE_OPTIONAL,
```

Comment:
Why did we change DIFFUSER code when we only were touching TRANSFORMERS? @Rexwang8